lib/jbuild

```diff
@@ -0,0 +1,11 @@
+(library
+ ((name ezxenstore)
+  (public_name ezxenstore)
+  (modules (xenstore watch))
+  (wrapped false)
```

Comment:
Jbuilder's documentation nicely explained it to me: https://jbuilder.readthedocs.io/en/latest/jbuild.html#library